docs/configuration.md

```diff
@@ -1,14 +1,15 @@
 # Configure the bot
 
-This page explains how to configure the bot.
+Freqtrade has many configurable options and possibilities.
+By default, these options are configured via the configuration file (see below).
```

Comment:
options --> features? (and in the line above)
in order to avoid confusion with the cli options...

Reply:
I've decided on "settings" for the 2nd line - not everything we configure is a feature ... (api-key is a setting, not a feature!!)